presto-hive/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -176,6 +177,50 @@ private static Properties getHiveSchema(
         return schema;
     }
 
+    public static Properties getHiveSchema(Map<String, String> serdeParameters, Map<String, String> tableParameters)
```

Comment:
Not quite sure understand this method.

Reply:
This is needed in the `BackgroundHiveSplitLoader` to avoid creating the whole schema, as it only needs custom properties from the storage descriptor with a possible override by table properties. The semantic is weird, but I'm just copying the existing behaviour.